scene/2d/tile_map.cpp

```diff
@@ -157,7 +157,13 @@ void TileMap::_update_quadrant_transform() {
 
 	Transform2D nav_rel;
 	if (bake_navigation) {
-		nav_rel = get_relative_transform_to_parent(navigation);
+		if (navigation) {
+			nav_rel = get_relative_transform_to_parent(navigation);
+		} else if (Object::cast_to<Node2D>(get_parent())) {
+			nav_rel = get_relative_transform_to_parent(get_parent());
```

Comment:
```suggestion
```

`get_transform()` returns the transform relative to the direct parent (result of `get_parent()`) so this is equivalent to the `else` block and thus it's not needed.